x-pack/plugins/ml/public/application/explorer/dashboard_controls/add_anomaly_charts_to_dashboard_controls.tsx

```diff
@@ -59,16 +63,19 @@ export const AddAnomalyChartsToDashboardControl: FC<AddToDashboardControlProps>
       };
     }
 
-    const config = getDefaultEmbeddablePanelConfig(jobIds);
+    const config = getDefaultEmbeddablePanelConfig(jobIds, queryString);
 
     return {
       ...config,
       jobIds,
       maxSeriesToPlot: maxSeriesToPlot ?? DEFAULT_MAX_SERIES_TO_PLOT,
       severityThreshold: severity.val,
-      ...(timeRange ?? {}),
+      ...(timeRange ? { timeRange } : {}),
+      ...(queryString !== undefined
+        ? { query: { query: queryString, language: SEARCH_QUERY_LANGUAGE.KUERY } as Query }
+        : {}),
     };
-  }, [selectedCells, interval, bounds, jobIds, maxSeriesToPlot, severity]);
+  }, [selectedCells, interval, bounds, jobIds, maxSeriesToPlot, severity, queryString]);
```

Comment:
Ad discussed, it would be nice if this same sort of behavior, where the charts embeddable is filtered by influencer field / value is applied when the user has selected a cell in the view by swim lane  (applies when adding to a case or a dashboard). Currently it only applies the filter when it is set in the Anomaly Explorer query bar. For example, here the embeddable in the dashboard / case should only show status code : 404

<img width="1397" alt="image" src="https://user-images.githubusercontent.com/7405507/189378712-ad4814a5-8166-4aa7-86c4-e6725c0a5c34.png">

Reply:
Updated here [`637eaa3` (#140147)](https://github.com/elastic/kibana/pull/140147/commits/637eaa3ad45965a9f5ef016d7942394f4ce2bd90). It will not apply the field value pair from the selected cells.